openhtf/io/output/json_factory.py

```diff
@@ -1,13 +1,13 @@
 """Module for outputting test record to JSON-formatted files."""
 
 import base64
+import output_to_file
 from json import JSONEncoder
-
 from openhtf.exe import test_state
 from openhtf.util import data
 
 
-class OutputToJSON(JSONEncoder):
+class OutputToJSON(JSONEncoder, output_to_file.OutputToFile):
```

Comment:
multiple inheritance can get quirky, better to avoid it when convenient.  I'd just create a JSONEncoder instance in `__init__` and save it on `self` for use in `__call__` (don't forget to pass **kwargs through to it's constructor).  This technique is referred to as 'composition' rather than inheritance: [more reading](https://www.thoughtworks.com/insights/blog/composition-vs-inheritance-how-choose)